proxy/certs/certs.go

```diff
@@ -317,23 +318,6 @@ func (s *RemoteCertSource) Remote(instance string) (cert *x509.Certificate, addr
 		return nil, "", "", "", err
 	}
 
-	// TODO(chowski): remove this when us-central is removed.
```

Comment:
Thinking some more about this, I think we might be OK with removing all this region check logic.

That said, there might be some clients providing `us-central` instead of `us-central1`. So I think the right thing to do here is: 1. remove all this region checking, 2. add a check just about the call to `s.serv.Connect.Get` such that any region of `us-central` is replaced with `us-central1`.